stream.go

```diff
@@ -100,14 +100,15 @@ func (s *StreamingSession) readNext() error {
 	}
 
 	// Timeout after 100ms (Maybe this needs to be changed?)
-	timeOut := time.After(time.Second)
+	timeOut := time.NewTimer(time.Second)
 
 	// This will attempt to send on the channel before the timeout, which is 1s
 	select {
-	case <-timeOut:
+	case <-timeOut.C:
 		return ErrVoiceConnClosed
 	case s.vc.OpusSend <- opus:
 	}
+	timeOut.Stop()
```

Comment:
maybe move it under the case where opus send succeeds

Reply:
Done!